src/DeviceListener.ts

```diff
@@ -385,12 +382,12 @@ export default class DeviceListener {
     }
 
     private checkKeyBackupStatus = async (): Promise<void> => {
-        if (this.keyBackupStatusChecked || !this.client) {
+        if (!this.client) {
             return;
         }
-        // returns null when key backup status hasn't finished being checked
-        const isKeyBackupEnabled = this.client.getKeyBackupEnabled();
-        this.keyBackupStatusChecked = isKeyBackupEnabled !== null;
+        // force a backup check
+        await this.client.getCrypto()?.checkKeyBackupAndEnable();
```

Comment:
why do we now enable backups where previously we did not?

Reply:
It's the only API we have now. I am not sure why it was seperate APIs previously, why would we check backup and not activate it?